portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,13 +88,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String lastName = getParameter(request, "last-name", "");
     String commentText = getParameter(request, "comment", "");
     Date commentDate = new Date();
+    String commentId = UUID.randomUUID().toString();
```

Comment:
I'm pretty sure datastore has methods available to generate IDs.

Reply:
I tried looking that up and also tried using the default ID/Name field but I could not access it.